ably/README.md

```diff
@@ -0,0 +1,65 @@
+## Overview
+The [Ably][1] platform is used to power realtime use cases such as multiplayer, chat, data synchronisation, data broadcast and notifications, for high-scale web and mobile applications around the world.
+
+Using our APIs, engineers are free to focus on building core functionality, rather than having to provision and maintain servers and cloud infrastructure.
```

Comment:
```suggestion
The [Ably][1] platform is used to power realtime use cases such as multiplayer, chat, data synchronisation, data broadcast and notifications, for high-scale web and mobile applications around the world. Using Ably's APIs, engineers are free to focus on building core functionality, rather than having to provision and maintain servers and cloud infrastructure.
```

Reply:
Updated in e20e3b965b44b64683573fa2cb3ca92045d2e600